lib/tldr/runner.rb

```diff
@@ -52,6 +53,7 @@ def run_test test, config, plan, reporter
       e = nil
       start_time = Process.clock_gettime(Process::CLOCK_MONOTONIC, :microsecond)
       wip_test = WIPTest.new(test, start_time)
+      wip_test.thread = Thread.current if config.verbose_cancelled_trace
```

Comment:
```suggestion
      wip_test = WIPTest.new(test, start_time, Thread.current)
```

I don't like the idea of conditionally mutating a value. If we sometimes need to track a thing that costs approximately nothing to track, then we should always track it.

Reply:
Ta! Have committed that suggestion =)